.github/workflows/lint.yml

```diff
@@ -16,9 +16,17 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: 3.7
+      - name: Install project dependencies
+        run: |
+          pip install tensorflow==2.0.0 colorama==0.4.1
+          pip install -e .
       - name: Install Lint Dependencies
-        run: pip install pyflakes==2.1.1 black==19.10b0
-      - name: Run PyFlakes
-        run: pyflakes .
+        run: pip install flake8==3.7.9 black==19.10b0 pytype==2019.10.17 isort==4.3.21
+      - name: Run flake8
+        run: flake8 --ignore=E203,E501,W503 --exclude build,dist
       - name: Black code style
-        run: black . --check --target-version py36 --exclude 'build/|buck-out/|dzzist/|_build/|\.git/|\.hg/|\.mypy_cache/|\.tox/|\.venv/|larq/snapshots/'
+        run: black . --check --target-version py37 --exclude 'build/|buck-out/|dzzist/|_build/|\.git/|\.hg/|\.mypy_cache/|\.tox/|\.venv/|larq/snapshots/'
+      - name: Check import order with isort
+        run: isort --check-only --use-parentheses --trailing-comma -m 3 -l 88 -rc zookeeper examples setup.py
```

Comment:
These rules in theory make `isort` compatible with `Black`.

Reply:
I'm not sure why it's failing on CI because the test works locally for me.... I'll try to investigate.